openshift/release/artifacts/eventing-core.yaml

```diff
@@ -1247,15 +1247,33 @@ metadata:
     app.kubernetes.io/version: v1.10
     app.kubernetes.io/name: knative-eventing
   annotations:
-    # TODO add schemas and descriptions
+    # TODO add schemas
     registry.knative.dev/eventTypes: |
       [
-        { "type": "dev.knative.apiserver.resource.add" },
-        { "type": "dev.knative.apiserver.resource.delete" },
-        { "type": "dev.knative.apiserver.resource.update" },
-        { "type": "dev.knative.apiserver.ref.add" },
-        { "type": "dev.knative.apiserver.ref.delete" },
-        { "type": "dev.knative.apiserver.ref.update" }
+        {
+          "type": "dev.knative.apiserver.resource.add",
```

Comment:
I guess these are the latest changes from upstream?

Reply:
No, we missed this codegen from a previous PR and codegen is not verified